### PR TITLE
Added changelog entries for OS X 10.11 fixes

### PR DESCRIPTION
**Pull request by natewalck**

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## GitHub Users Who Are Not Authorized To Contribute

The following GitHub users do not appear to have signed a CLA:

* @natewalck

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Comment by natewalck** [comment]
@jaymzh for the merge

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,8 +17,9 @@
   [pr#3771](https://github.com/chef/chef/pull/3771) add depth property for deploy resource
 * [**James Belchamber**](https://github.com/JamesBelchamber):
   [pr#1796](https://github.com/chef/chef/pull/1796): make mount options aware
-
-
+* [**Nate Walck**](https://github.com/natewalck):
+ * [pr#3594](https://github.com/chef/chef/pull/3594): update service provider for OSX 10.11
```
Comment (jaymzh) [inline CHANGELOG.md]: not the format in the rest of the file, no secondary '*'s for these

**Comment by natewalck** [comment]
Should I put my name 2x or just put the commits down without my name? I don't care either way ;)

**Comment by jaymzh** [comment]
```
* name:
pr
* name:
pr
```

**Comment by natewalck** [comment]
Fixed the formatting.

Also, I thought Markdown was supposed to be consistent...I can't get that awkward extra line gone.

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,7 +17,10 @@
   [pr#3771](https://github.com/chef/chef/pull/3771) add depth property for deploy resource
 * [**James Belchamber**](https://github.com/JamesBelchamber):
   [pr#1796](https://github.com/chef/chef/pull/1796): make mount options aware
-
+* [**Nate Walck**](https://github.com/natewalck):
+* [pr#3594](https://github.com/chef/chef/pull/3594): update service provider for OSX 10.11
```
Comment (jaymzh) [inline CHANGELOG.md]: remove the star

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,7 +17,10 @@
...
+* [**Nate Walck**](https://github.com/natewalck):
+* [pr#3704](https://github.com/chef/chef/pull/3704): Add SIP (OS X 10.11) support
```
Comment (jaymzh) [inline CHANGELOG.md]: remove the star.

File: CHANGELOG.md [hunk 4]
```diff
@@ -17,7 +17,10 @@
   [pr#3771](https://github.com/chef/chef/pull/3771) add depth property for deploy resource
 * [**James Belchamber**](https://github.com/JamesBelchamber):
```
Comment (jaymzh) [inline CHANGELOG.md]: here is your positive example.

**Comment by natewalck** [comment]
I tried several different incantations. I cannot please the Markdown god(s).

**Comment by jaymzh** [comment]
Fugedaboutit. Done here: https://github.com/chef/chef/pull/3800 :)

**Comment by natewalck** [comment]
Thanks! :+1:

**Pull request closed without merging**